app/assets/stylesheets/components/helpers/_variables.scss

```diff
@@ -20,6 +20,5 @@ $app-border-width-form-element: 2px;
 
 // Focus
 $app-focus-width: 3px;
-$app-focus-colour: $focus-colour;
 
 $app-error-colour: $red;
```

Comment:
Where does `$red` come from? Could we replace `$app-error-colour` with a govuk colour, directly in declarations rather than abstracting it out like this? Would make the code a bit less tunnelly.

Reply:
No idea, but we didn't introduce this in this PR.